### PR TITLE
Enabling the first cloud test

### PR DESCRIPTION
**Pull request by jean-philippe-martin**
This one passes for me locally. Let's see what Travis says.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -195,9 +195,12 @@ test {
...
     useTestNG{
         if ( "$System.env.CLOUD" =="true"){
-            includeGroups 'cloud', 'bucket'
+            // run everything
+            includeGroups 'cloud', 'bucket', 'cloud_todo', 'bucket_todo'
         } else {
-            excludeGroups 'cloud', 'bucket'
+            // run everything except the "todo" tests
+            excludeGroups 'cloud_todo', 'bucket_todo'
+            includeGroups 'cloud', 'bucket'
```
Comment (lbergelson) [inline build.gradle]: Include excludes anything that isn't explicitly included.  So you're accidentally excluding all tests that are not cloud or bucket.   If you look at the [test log](https://storage.googleapis.com/hellbender/test/build_reports/1840.2/tests/index.html) you'll see that you're actually not running any tests now in the standard run.

**Comment by lbergelson** [comment]
We can see which cloud tests are already passing and put them in the "cloud" or "bucket' group instead of the todo.

https://storage.googleapis.com/hellbender/test/build_reports/1840.1/tests/index.html

**Comment by jean-philippe-martin** [comment]
Oh good, I'll use that list for the next PR. Hoping to start getting LGTMs so I don't have so many in flight.

File: build.gradle (hunk 1 above)
Comment (jean-philippe-martin) [inline build.gradle]: Indeed. I discovered that in the meantime (why didn't you review #819 before #820?). I've fixed that in #822.

Comment (lbergelson) [inline build.gradle]: Heh, not sure why I'm out of order.

**Comment by jean-philippe-martin** [comment]
this is superseded by #823 , so closing.

**Pull request closed without merging**